File: src/main/java/com/dude/dms/ui/converters/LocalDateTimeConverter.java
package com.dude.dms.ui.converters;

import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

import static com.dude.dms.backend.brain.OptionKey.DATE_FORMAT;

public final class LocalDateTimeConverter {

    private LocalDateTimeConverter() {
    }

    public static String convert(LocalDateTime localDateTime) {
        return localDateTime == null ? null : localDateTime.format(DateTimeFormatter.ofPattern(DATE_FORMAT.getString() + " HH:mm:ss"));
    }
}

File: src/main/java/com/dude/dms/ui/converters/LocalDateTimeConverter.kt
package com.dude.dms.ui.converters

import com.dude.dms.backend.brain.OptionKey
import java.time.LocalDateTime
import java.time.format.DateTimeFormatter

fun LocalDateTime.convert() = format(DateTimeFormatter.ofPattern("${OptionKey.DATE_FORMAT.string} HH:mm:ss"))